pwndbg/heap/ptmalloc.py

```diff
@@ -662,7 +658,18 @@ def initialize_structs(method):
     def wrapper(self, *args, **kwargs):
         # If structs module for HeuristicHeap is None, try to initialize it.
         if self._structs_module is None:
-            self._structs_module = importlib.reload(importlib.import_module('pwndbg.heap.structs'))
+            try:
+                self._structs_module = importlib.reload(importlib.import_module('pwndbg.heap.structs'))
+            except TypeError as e:
+                print(
+                    message.hint(
+                        "There is a TypeError when importing the structs for heap heuristic.\n"
```

Comment:
Do users care if its TypeError or other error? :P

Reply:
Ahh, maybe the user won't care xD
I'll modify the description of this lol